src/hooks.server.ts

```diff
@@ -221,7 +221,8 @@ const handleGlobal: Handle = async ({ event, resolve }) => {
 						_id: user._id,
 						login: user.login ? user.login : '',
 						roleId: user.roleId,
-						alias: user.alias
+						alias: user.alias,
+						hasPOSOptions: user.hasPOSOptions
```

Comment:
Since "any" user can now have access to the POS options; it's redundant to keep the `POS_ROLE_ID` role.

I think we should either remove `POS_ROLE_ID` in favor of `hasPOSOptions` or replace `hasPOSOptions` with a method `canAccessPosOptions()` that takes care of doing the projection, regardless of how the user gained the capability.